package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "doi-ui",
```

Comment:
The (new) standard is to name the packages with pds- prefix, but have the repo name without the pds- prefix (the opposite of waht we usually do).
For now can you just rename the package pds-doi-ui.

Reply:
Got it, I'll make this change.